TradingSystem/StockBrockerDriver.cpp

```diff
@@ -1,2 +1,57 @@
 #include "StockDriver.h"
+#include <string>
 
+using std::string;
+
+class KiwerStockDriver : public StockDriver {
+	// StockDriver��(��) ���� ��ӵ�
+	bool login(int id, int pass) override
+	{
+
+
+		return false;
+	}
+	void buy(int code, int price, int count) override
+	{
+	}
+	void sell(int code, int price, int count) override
+	{
+	}
+	int getPrice(int code, time_t time) override
+	{
+		return 0;
+	}
+};
+
+class NemoStockDriver : public  StockDriver{
+	// StockDriver��(��) ���� ��ӵ�
+	bool login(int id, int pass) override
+	{
+		return false;
+	}
+	void buy(int code, int price, int count) override
+	{
+	}
+	void sell(int code, int price, int count) override
+	{
+	}
+	int getPrice(int code, time_t time) override
+	{
+		return 0;
+	}
+};
+
+class StockBrockerDriver {
+private:
+	StockDriver* userStock;
+
+public:
+
+	StockDriver* selectStockBrocker(string stock)
+	{
+		if (stock == "Kiwer") return new KiwerStockDriver();
+		if (stock == "Nemo") return new NemoStockDriver();
+		else return nullptr;
+	}
```

Comment:
오우 factory로직이 잘 구현된거 같습니다
다만 이렇게 하면 mocking으로 test를 할수 없으니
아래와 같이 고쳐 보면 어떨까요??

```suggestion
	void selectStockBrocker(string stock)
	{
		if (stock == "Kiwer") current_stock = kiwer_stock;
		if (stock == "Nemo") current_stock = nemo_stock;
		return;
	}
	
private:
    StockDriver* currnet_stock
    StockDriver* nemo_stock
    StockDriver* kiwer_stock
```

Reply:
네 제안주신부분 확인해서, 의도에 부합하도록 재수정 하였습니다.